### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized control type property is wrong for "Edit" link

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#45]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1. Open https://code.visualstudio.com/docs/introvideos/overview in EDGE.
2. navigate to the "Edit" link available beside "Introductory Videos" Header
3. Observe the "Localized Control Type" property in Inspect

**Same issue is also observed with the link: (in all the links "Edit" is available beside the Navigation landmarks on the RHS of the page)**
https://code.visualstudio.com/docs/setup/setup-overview
https://code.visualstudio.com/docs/editor/codebasics
https://code.visualstudio.com/docs/customization/userandworkspace
https://code.visualstudio.com/docs/runtimes/nodejs
https://code.visualstudio.com/docs/languages/overview
https://code.visualstudio.com/docs/supporting/faq
https://code.visualstudio.com/docs/editor/accessibility

**Actual Result:**
The "Localized Control Type" property is button instead of link.

**Narrator Behaviour:**
Narrator announces as "Edit Button"

**Expected Result:**
The Localized Control Type property should be link

**User Impact:**
If the control type is button, it will be confusing for a screen reader user and the user will understand that the edit window opens in the same page.

**Screenshots:**
![localized control type of edit link is button instead of link](https://cloud.githubusercontent.com/assets/24825441/21718124/288f1068-d43b-11e6-9ec0-09f83fec3b5a.png)

**Comment by waderyan** [comment]
Fixed. 
![image](https://cloud.githubusercontent.com/assets/2449568/26330682/fe476102-3f01-11e7-8435-247a7cc98948.png)
